build/loc.proj

```diff
@@ -73,6 +72,16 @@
     <Copy SourceFiles="@(EulaFiles->'%(Identity)')" DestinationFiles="@(EulaFiles->'%(DestinationDir)')"/>
   </Target>
 
+  <Target Name="MoveLcgFilesToArtifacts" AfterTargets="Localize" Condition="'$(BuildRTM)' != 'true'">
+    <ItemGroup>
+      <_MoveLcgFiles Include="@(LocalizedToolFiles)" Condition="'%(LocalizedToolFiles.Extension)'== '.lcg'">
+        <DestinationDir>$(ArtifactsDirectory)localize\PLOC\net\%(LocalizedToolFiles.Filename)%(LocalizedToolFiles.Extension)</DestinationDir>
```

Comment:
yes, let's not worry about what pattern WSR-TC was following and go with the new one where all lcg files go under their respective projects. We will update the instructions for vendors to look for appearance of new project folders instead of just lcg files

also, instead of dropping these files to the artifacts folder in the build drop, consider creating a new location which just has lcg files. something like \\nuget\nuget\share\drops\ci\nuget.client\PLOC\build-4099